packages/widgets/src/lib/components/InferenceWidget/shared/WidgetHeader/WidgetHeader.svelte

```diff
@@ -53,7 +53,7 @@
 			<div class="flex items-center text-lg">
 				{#if !isDisabled}
 					<IconLightning classNames="-ml-1 mr-1 text-yellow-500" />
-					Inference Endpoints (serverless)
+					Inference API (serverless)
```

Comment:
```suggestion
					Inference API
```
i would omit the (serverless) here for aesthetics